examples/step-48/step-48.cc

```diff
@@ -109,30 +109,29 @@ namespace Step48
   // mass matrix. Since we use Gauss-Lobatto elements, the mass matrix is a
   // diagonal matrix and can be stored as a vector. The computation of the
   // mass matrix diagonal is simple to achieve with the data structures
-  // provided by FEEvaluation: Just loop over all (macro-) cells and integrate
-  // over the function that is constant one on all quadrature points by using
-  // the <code>integrate</code> function with @p true argument at the slot for
-  // values. Finally, we invert the diagonal entries since we have to multiply
-  // by the inverse mass matrix in each time step.
+  // provided by FEEvaluation: Just loop over all cell batches, i.e.,
+  // collections of cells due to SIMD vectorization, and integrate over the
+  // function that is constant one on all quadrature points by using the
+  // <code>integrate</code> function with @p true argument at the slot for
+  // values. Finally, we invert the diagonal entries to have the inverse mass
+  // matrix directly available in each time step.
   template <int dim, int fe_degree>
   SineGordonOperation<dim, fe_degree>::SineGordonOperation(
     const MatrixFree<dim, double> &data_in,
     const double                   time_step)
     : data(data_in)
     , delta_t_sqr(make_vectorized_array(time_step * time_step))
   {
-    VectorizedArray<double> one = make_vectorized_array(1.);
-
     data.initialize_dof_vector(inv_mass_matrix);
 
     FEEvaluation<dim, fe_degree> fe_eval(data);
     const unsigned int           n_q_points = fe_eval.n_q_points;
 
-    for (unsigned int cell = 0; cell < data.n_macro_cells(); ++cell)
+    for (unsigned int cell = 0; cell < data.n_cell_batches(); ++cell)
```

Comment:
Are these three same?

Reply:
yes, but the new variant is more descriptive, so I prefer it here.